itenium-socks/src/app/pages/why-us.component.ts

```diff
@@ -1,10 +1,15 @@
 import { Component } from '@angular/core';
 import { RouterLink } from '@angular/router';
+import {Title} from "@angular/platform-browser";
 
 @Component({
   selector: 'app-why-us',
   standalone: true,
   imports: [RouterLink],
   templateUrl: './why-us.component.html'
 })
-export class WhyUsComponent {}
+export class WhyUsComponent {
+  constructor(private titleService: Title) {
+    this.titleService.setTitle("Why Us | Toe-tally Amazing");
```

Comment:
Yes!

Enige probleem is nu... Als je ooit op de "Why Us" page kwam, dan blijft de title eeuwig dat!

Reply:
Geimplementeerd zoals beschreven in de acceptatiecriteria!
Indien ze effectief enkel op de Why Us pagina die titel willen en elders de originele is er deze MR: https://github.com/itenium-be/Angular-Days/pull/31

Reply:
Hahaha!
Of de analysts / testers er zo ook over gaan denken... 😀